packages/teams-js/src/public/media.ts

```diff
@@ -434,27 +434,31 @@ export namespace media {
     protected notifyEventToHost(mediaEvent: MediaControllerEvent, callback?: (err?: SdkError) => void): Promise<void> {
       ensureInitialized(FrameContexts.content, FrameContexts.task);
 
-      return Promise.resolve()
-        .then(() => {
-          throwExceptionIfMobileApiIsNotSupported(nonFullScreenVideoModeAPISupportVersion);
-        })
-        .then(() => {
-          const params: MediaControllerParam = { mediaType: this.getMediaType(), mediaControllerEvent: mediaEvent };
+      try {
+        throwExceptionIfMobileApiIsNotSupported(nonFullScreenVideoModeAPISupportVersion);
+      } catch (err) {
+        const wrappedRejectedErrorFn: InputFunction<void> = () => Promise.reject(err);
 
+        return callCallbackWithSdkErrorFromPromiseAndReturnPromise(wrappedRejectedErrorFn, callback);
```

Comment:
I think if we hit this line, then callback won't be passed the SdkError (looking at line 233 of utils.callCallbackWithSdkErrorFromPromiseAndReturnPromise). If I'm right, we could do something like wrap callback and pass it in like so:

const wrappedCallback = () => callback(err);
return callCallbackWithSdkErrorFromPromiseAndReturnPromise(wrappedRejectedErrorFn, wrappedCallback);

Reply:
We spoke about this offline, in the utility function, the error that is passed into the callback is the one that you receive from the rejected promise, if I write it the way you suggested, it will, still work, but i'll have to add an additional check to make sure that callback is defined in the cases where the callback is not provided.  I think this is extra code that isn't necessary

Reply:
Yes, agree, we spoke and that makes sense